### ISSUE TITLE
Uncaught TypeError: Invalid Point: (NaN, 0)

### ISSUE THREAD

**Issue by bflyblue** [atom-haskell/ide-haskell#192]
1. Have multiple Haskell files open
2. Drag one to the right to split the view into two

**Atom**: 1.14.3 x64
**Electron**: 1.3.13
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [ide-haskell](https://github.com/atom-haskell/ide-haskell) package 1.9.3

### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, 0)

```
At C:\Users\shaun\AppData\Local\atom\app-1.14.3\resources\app.asar\node_modules\text-buffer\lib\point.js:63

TypeError: Invalid Point: (NaN, 0)
    at Function.module.exports.Point.assertValid (/app.asar/node_modules/text-buffer/lib/point.js:63:15)
    at DisplayLayer.translateScreenPosition (/app.asar/node_modules/text-buffer/lib/display-layer.js:326:11)
    at TextEditor.module.exports.TextEditor.bufferPositionForScreenPosition (/app.asar/src/text-editor.js:1624:32)
    at bufferPositionFromMouseEvent (/packages/ide-haskell/lib/utils.coffee:12:12)
    at /packages/ide-haskell/lib/editor-control.coffee:48:18)
    at HTMLDivElement.dispatch (/packages/ide-haskell/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (/packages/ide-haskell/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
     -8:52.2.0 vim-mode-plus:move-to-last-character-of-line (input.hidden-input)
     -8:50.2.0 vim-mode-plus:move-down (input.hidden-input)
     -8:49.4.0 core:save (input.hidden-input)
 12x -8:30.2.0 vim-mode-plus:move-down (input.hidden-input)
  2x -8:28.4.0 vim-mode-plus:move-up (input.hidden-input)
 42x -8:26.9.0 vim-mode-plus:move-down (input.hidden-input)
 38x -8:23.8.0 vim-mode-plus:move-up (input.hidden-input)
  2x -7:13.8.0 vim-mode-plus:move-left (input.hidden-input)
     -7:13.5.0 vim-mode-plus:insert-after (input.hidden-input)
     -7:11.1.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
     -7:10.6.0 vim-mode-plus:insert-after (input.hidden-input)
     -7:10.2.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
     -7:09.8.0 core:save (input.hidden-input)
     -3:53.3.0 ide-haskell:close-tooltip (input.hidden-input)
  2x -3:07.0 tree-view:remove (span.name.icon.haskell-icon.medium-purple)
     -2:51 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-material-syntax 1.0.2 
atom-material-syntax-dark 0.2.7 
atom-material-syntax-light 0.4.6 
atom-material-ui 1.3.9 
autocomplete-glsl 0.2.3 
autocomplete-haskell 0.7.2 
blame 0.10.2 
dash 1.7.0 
duotone-dark-forest-syntax 2.1.0 
duotone-dark-sea-syntax 2.1.0 
duotone-dark-space-syntax 2.1.0 
duotone-dark-syntax 2.1.0 
duotone-light-syntax 2.1.0 
ficus-dark-syntax 2.0.0 
ficus-syntax 2.0.0 
file-icons 2.0.15 
flatwhite-syntax 0.8.0 
fonts 2.0.0 
git-diff-details 1.4.0 
git-log 0.4.1 
git-plus 7.3.1 
haskell-ghc-mod 1.19.7 
ide-haskell 1.9.3 
ide-haskell-hasktags 0.0.10 
isotope-light-ui 2.2.6 
isotope-ui 2.7.1 
language-glsl 2.0.1 
language-haskell 1.12.1 
linter 1.11.23 
linter-glsl 2.1.1 
markdown-preview-enhanced 0.9.11 
markdown-preview-plus 2.4.3 
merge-conflicts 1.4.4 
multi-cursor 2.1.5 
nord-atom-syntax 0.6.0 
nord-atom-ui 0.10.1 
pandoc 0.2.2 
spacegray-atom-dark-syntax 1.3.2 
spacegray-dark-ui 0.12.0 
sync-settings 0.8.1 
tab-numbers 0.6.0 
todo-show 1.11.0 
tree-view-git-status 1.3.1 
vim-mode-plus 0.82.3 
Zen 0.16.4 
```

**Comment by lierdakil** [comment]
Should be fixed with 1.19.7. Thanks for reporting!

**Comment by bflyblue** [comment]
Thanks for fixing this so promptly! Can confirm it's fixed for me.
